Gemfile

```diff
@@ -13,7 +13,8 @@ gem "unicorn-worker-killer", "0.4.4"
 
 # Federation
 
-gem "diaspora_federation-rails", "0.2.0"
+gem "diaspora_federation-json_schema", "0.2.0", github: "diaspora/diaspora_federation", branch: "develop"
```

Comment:
I don't mind how you split the commits, as long as it makes sense and commit messages match, it's OK for me. I would probably do the version-bump in a separate commit and add the additional gem together in the commit where you use it.